index.html

```diff
@@ -0,0 +1,226 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      crossorigin="anonymous"
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@400&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@700&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./style.css" />
+    <title>Af Esthétique Beauty Therapist</title>
+  </head>
+
+  <body>
+    <header class="desk">
+      <div class="header1">
+        <div class="header1-contents">
+          <a href="https://web.facebook.com"
+            ><i class="fa fa-facebook" aria-hidden="true"></i
+          ></a>
+          <a href="https://twitter.com"
+            ><i class="fa fa-twitter" aria-hidden="true"></i
+          ></a>
+          <ul class="lang-page-log">
+            <li><button type="button">English</button></li>
+            <li><button type="button">My page</button></li>
+            <li><button type="button">Log out</button></li>
+          </ul>
+        </div>
+      </div>
+      <nav>
+        <a href="./index.html" class="logo"
+          ><img class="logo" src="images/logo.png" alt="logo"
+        /></a>
+        <a class="nav-links-desk" href="./index.html">Home Service</a>
+        <a class="nav-links-desk" href="./about.html">About Af</a>
+      </nav>
+    </header>
+    <header class="mob">
+      <a class="ham">
+        <div id="toggle-icon">
+          <div class="ham-bar1"></div>
+          <div class="ham-bar2"></div>
+          <div class="ham-bar3"></div>
+        </div>
+      </a>
+      <nav class="header2">
+        <div>
+          <a href="./index.html">
+            <h2 class="header2-content-mob">Home Service</h2>
+          </a>
+        </div>
+        <div>
+          <a href="./about.html">
+            <h2 class="header2-content-mob">About Af</h2>
+          </a>
+        </div>
+      </nav>
+    </header>
+    <section class="pink-section">
+      <section>
+        <div class="banner-desk">
+          <img src="images/massage-bg.png" class="bg-img" alt="massage" />
+        </div>
```

Comment:
- [ ] This image is making your design different and not that much effective. I recommend using this as an background image and text over it. By this you will follow the design and it will make your website more awesome. Also try to keep the font-size similar. :smile: 

### Desktop version

### Your design 

![Screenshot from 2021-07-09 20-45-10](https://user-images.githubusercontent.com/22434660/125100678-0166f280-e0f7-11eb-82ef-3e54c6be8772.png)


### Behance design
 
![Screenshot from 2021-07-09 20-45-35](https://user-images.githubusercontent.com/22434660/125100698-04fa7980-e0f7-11eb-9071-ec48837db349.png)